docs/client/admin.rst

```diff
@@ -599,3 +599,73 @@ Optional
 .. cmdoption:: -o, --output
 
     Output the results into a json file
+
+
+-------
+
+Pull a Synapse Project's Wiki
+-----------------------------
+
+Synopsis
+^^^^^^^^
+
+pull-wiki
+    projectid [--workdir]
+
+Description
+^^^^^^^^^^^
+
+Download each wiki page of a synapse project and saves a json configuration
```

Comment:
Maybe include "Download each wiki page of a synapse project as individual .md files"?